roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -1046,73 +1126,117 @@ signing_key_path: "/data/{{ matrix_server_fqn_matrix }}.signing.key"
 #      "ed25519:auto": "abcdefghijklmnopqrstuvwxyzabcdefghijklmopqr"
 #  - server_name: "my_other_trusted_server.example.com"
 #
-# The default configuration is:
+trusted_key_servers: {{ matrix_synapse_trusted_key_servers|to_json }}
```

Comment:
This probably needs a note in the CHANGELOG. It isn't new that the default is matrix.org but it is newly configurable so people may not want to use matrix.org for privacy reasons.